### PR TITLE
make the dropdown red scroll

### PR DESCRIPTION
**Pull request by kol4k**
demo [https://pedantic-dubinsky-7a8c94.netlify.com/index.html](https://pedantic-dubinsky-7a8c94.netlify.com/index.html)
captureitplus636485124706873364

### CODE REVIEW

**Comment by anshumanv** [comment]
@kaine119 looks nice, can you please try to exclude commits which are not related to this PR?
Thanks :smile:

**Comment by kol4k** [comment]
how i can do it? @anshumanv

**Comment by anshumanv** [comment]
You need to update your fork with the upstream. Follow - https://help.github.com/articles/syncing-a-fork/
Alternatively, you can google about syncing your fork on github.

**Comment by kol4k** [comment]
done, i've been fetch upstream.

**Comment by kol4k** [comment]
i've been snyc fork. thank you @anshumanv

**Comment by anshumanv** [comment]
Ideally, the commits should go which are not a part of this PR, can you try dropping those commits from this PR while rebasing?

**Comment by anshumanv** [comment]
Nice! @kol4k can you try squashing those commits to make things neat?

**Comment by kol4k** [comment]
@anshumanv when i squash i got notice this:
$ git rebase -i HEAD~3
Cannot rebase: You have unstaged changes.
Please commit or stash them.

**Comment by anshumanv** [comment]
Did you make any changes after making this PR, If yes, `$ git stash` to remove them, it's nice to see you try squashing commits and it's purely for learning, if unable we can squash commits on our end while merging. :smile:

**Comment by kol4k** [comment]
no i did not make changes after this pr. https://github.com/fossasia/gci17.fossasia.org/pull/229/commits/89331b5531c2ce411a1fe14b1512b1d0541bc9b1

File: _includes/header.html [hunk 1]
```diff
@@ -63,28 +63,33 @@
...
 		                                <li>
-		                                    <a href="#">
-		                                        Events
-		                                    </a>
-		                                </li>
-		                                <li>
-		                                    <a href="https://gci16.fossasia.org">
-		                                        Google Code-In 2016
-		                                    </a>
+											<a href="#">Events</a>
```
Comment (anshumanv) [inline _includes/header.html]: Please correct the indentation of this block.

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 1 above)
Comment (kol4k) [reply _includes/header.html]: I just add multi level menu for event list

**Review by isuruAb — APPROVED** [review]
👍  LGTM .

**Review by anshumanv — APPROVED** [review]

**Review by ParthS007 — APPROVED** [review]
LGTM !

**Comment by kol4k** [comment]
Thank you so much 😊

**Comment by anshumanv** [comment]
@kol4k can you please check why they it's not appearing on the actual website even after this has been merged?

Your deployment link doesn't show proper content either.

**Comment by ParthS007** [comment]
@anshumanv Please check again, It's working for me!

**Comment by anshumanv** [comment]
@ParthS007 not sure then what's the problem in my case, are you sure about it?
image

**Comment by kol4k** [comment]
Yes, I am sure. because task is to make red color on scroll and I make multi dropdown menu because some things to be more easily categorized. @anshumanv

**Comment by anshumanv** [comment]
Thanks okay :sweat_smile: but can you thing of reasons why I'm not seeing red color in my case?

**Comment by kol4k** [comment]
In me it's work. 
screenshot_20171212_183008

**Comment by anshumanv** [comment]
Ah, no issues then @kol4k, there must be some problem in my case. 
Good going.

**Comment by the-codingninja** [comment]
It doesn't works for me too now also

**Comment by kol4k** [comment]
i test on different android phone.
screenshot_2017-12-12-23-36-06-08

**Pull request merged**